src/mlpack/methods/reinforcement_learning/environment/env_type.hpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file methods/reinforcement_learning/environment/env_type.hpp
+ * @author Nishant Kumar
+ *
+ * This file defines a dummy environment to be use with gym_tcp_api.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_RL_ENVIRONMENT_ENV_TYPE_HPP
+#define MLPACK_METHODS_RL_ENVIRONMENT_ENV_TYPE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace rl {
+
```

Comment:
Might be a good idea to add a simple example here, what do you think?

Reply:
Yeah. I've made the changes. Kindly let me know if something is missing.